index.bs

```diff
@@ -12349,12 +12363,54 @@ It is the responsibility of specifications using Web IDL to state
 which global environment (or, by proxy, which global object) each platform
 object is associated with.
 
+<div algorithm>
+  To create a <dfn export>new</dfn> object implementing the interface |interface| in the
+  Realm |realm|, with optional `new.target` value |newTarget|, perform the following steps:
```

Comment:
I think we should not allow providing `new.target`. I cannot think of any specs that would use it.